src/api/ActivitiesApi.ts

```diff
@@ -156,7 +156,7 @@ export default class ActivitiesApi implements IActivityApi {
       conditions.push(`<Geq><FieldRef Name='WeekOf'/><Value Type='DateTime'>${startDate.subtract(1, 'day').toISOString()}</Value></Geq>`);
     }
     if (endDate) {
-      conditions.push(`<Leq><FieldRef Name='WeekOf'/><Value Type='DateTime'>${endDate.toISOString()}</Value></Leq>`);
+      conditions.push(`<Leq><FieldRef Name='WeekOf'/><Value Type='DateTime'>${endDate.startOf('week').add(1, 'day').toISOString()}</Value></Leq>`);
```

Comment:
Add a comment as to why we're adding 1 day so we don't accidentally "fix" this in the future